visualizer/style/semanticsEditor.css

```diff
@@ -209,3 +217,32 @@ headerBlock real {
   color: #999;
 }
 
+/* `Force Evaluation` entry in `parseTreeMenu` */
+.contextMenu label span {
+  margin-left: 10px;
+}
+.contextMenu li.disabled ul {
```

Comment:
Disabled items shouldn't be hidden, they should be greyed out. If you need to hide an item, just don't add it to the menu when you're handling the `contextMenu` event.